spec/build/script/csharp_spec.rb

```diff
@@ -25,6 +25,22 @@
     end
   end
 
+  describe 'version switching' do
+    it 'throws a error with a invalid version' do
+      data[:config][:mono] = 'foo'
+      should include_sexp [:echo, 'foo is not a valid version of mono.', {:ansi=>:red}]
```

Comment:
Nah, sorry, I meant to enclose the version part of the string so the message reads `"foo" is not a valid version` on the console ;)